apps/site/assets/ts/stop/components/StationInformation.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement } from "react";
+import { Stop } from "../../__v3api";
+import ExternalMapLink from "./ExternalMapLink";
+
+const StationInformation = ({
+  stop
+}: {
+  stop: Stop;
+}): ReactElement<HTMLElement> => {
+  return (
+    <div>
+      <div>Station information PLACEHOLDER</div>
+      {/* TODO combine station name with municiplaity (station name, municiplaity) if no address */}
+      {stop.address && (
```

Comment:
**issue:** By checking for the address here, `ExternalMapLink` never gets to exercise its fallback logic of showing the municipality. So nothing shows up at all for, for example, Southampton St @ Newmarket St on the 171 bus.